src/app/qgsmaptooltrimextendfeature.h

```diff
@@ -0,0 +1,78 @@
+/***************************************************************************
+    qgmaptooltrimextendfeature.h  -  map tool to trim or extend feature
+    ---------------------
+    begin                : October 2018
+    copyright            : (C) 2018 by Loïc Bartoletti
+    email                : loic dot bartoletti at oslandia dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 3 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSMAPTOOLTRIMEXTENDFEATURE_H
+#define QGSMAPTOOLTRIMEXTENDFEATURE_H
+
+#include "qgsmaptooledit.h"
+#include "qgis_app.h"
+#include "qgspointlocator.h"
```

Comment:
this include doesn't seem to be required in header

Reply:
Oops, an unsuccessful trial residue, I delete.